packages/reports/addon/components/navi-date-picker.js

```diff
@@ -9,153 +9,145 @@
  *      onUpdate=(action handleUpdate)
  *   }}
  */
-import { scheduleOnce, next } from '@ember/runloop';
 import Component from '@ember/component';
-import { set, get, computed } from '@ember/object';
+import { get, set, computed, action } from '@ember/object';
+import { layout as templateLayout, classNames, className } from '@ember-decorators/component';
 import layout from '../templates/components/navi-date-picker';
 import moment from 'moment';
-import { getFirstDayEpochIsoDateTimePeriod, getIsoDateTimePeriod } from 'navi-core/utils/date';
+import range from 'lodash/range';
+import {
+  getFirstDayEpochIsoDateTimePeriod,
+  getFirstDayOfIsoDateTimePeriod,
+  getLastDayOfIsoDateTimePeriod,
+  getIsoDateTimePeriod
+} from 'navi-core/utils/date';
+
+@templateLayout(layout)
+@classNames('navi-date-picker')
```

Comment:
Are we accessing the element attribute of this component anywhere? If not, can we make this tagless?